app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb

```diff
@@ -35,9 +35,9 @@
       heading: "Engineering the future",
       colour: "pink",
       image_1_path: "static/images/content/hero-images/0017.jpg",
-      image_1_alt: "A teacher sitting with a student helping them with a maths problem",
+      image_1_alt: "",
       image_2_path: "static/images/content/hero-images/0004.jpg",
-      image_2_alt: "A maths teacher standing at the front of a full classroom delivering a lesson",
+      image_2_alt: "",
     ) do %>
       <p>
         Engineers and physicists help find solutions to some of our biggest problems. From climate change to next generation prosthetics, we need their problem-solving skills and creative thinking to help us survive some of the biggest challenges humanity has ever faced.
```

Comment:
@caitlin-reidy The copy in the box is quite a long chunk. Is it worth us putting a paragraph break after the first sentence?